def check_failed(file, line):
	return robocaller("check_failed", "void", file, line)

